sdks/wasm/Makefile

```diff
@@ -253,11 +251,11 @@ package: build build-dbg-proxy build-framework-nuget
 	rm tmp/release/.stamp-build
 	cp bindings.cs tmp
 	mkdir tmp/framework
-	cp WebAssembly.Bindings.dll tmp/framework
-	cp WebAssembly.Bindings.pdb tmp/framework
+	cp $(TOP)/sdks/out/wasm-bcl/wasm/WebAssembly.Bindings.dll tmp/framework
+	cp $(TOP)/sdks/out/wasm-bcl/wasm/WebAssembly.Bindings.pdb tmp/framework
 	cp WasmHttpMessageHandler.cs tmp
```

Comment:
Do these still need to be copied, since they are in the bcl now ?

Reply:
Left those there so as not break existing projects that use the links.